node_modules/ioslib/lib/certs.js

```diff
@@ -166,7 +166,7 @@ function detect(options, callback) {
 								if (!code) {
 									var tmp = [];
 									parseCerts(out, tmp);
-									results.certs.wwdr = tmp.length && tmp[0].invalid === false;
+									results.certs.wwdr = results.certs.wwdr || (tmp.length && tmp[0].invalid === false);
```

Comment:
`results.certs.wwdr` will never be truthy because if it was, the parent function would return on line 163.